inventree_brother/brother_plugin.py

```diff
@@ -47,6 +50,26 @@ def get_rotation_choices():
     return [(f"{degree}", f"{degree}°") for degree in [0, 90, 180, 270]]
 
 
+class BrotherLabelSerializer(serializers.Serializer):
+    """Custom serializer class for BrotherLabelPlugin.
+
+    Used to specify printing parameters at runtime
+    """
+
+    class Meta:
```

Comment:
Comment: the serializer also worked without a meta class in my tests, I guess it is only necessary for model fields.

Reply:
Yeah I saw your example now, I might simplify it